src/audio/sound_buffer.rs

```diff
@@ -99,6 +100,11 @@ impl SoundBufferRef {
         unsafe { ffi::sfSoundBuffer_getSampleCount(self.raw()) as i64 }
     }
 
+    /// Get the samples stored in the buffer
+    pub fn get_samples(&self) -> &[i16] {
```

Comment:
The current naming convention is to omit the `get` part where it makes sense. That would make this function named just `samples`.

Reply:
I would also like that better... I just used get_samples because the sample_count documentation talked about it.